pokecli.py

```diff
@@ -69,23 +64,22 @@ def main():
 
     colorama.init()
 
-    config = init_config()
-    if not config:
+    config_dir = init_config()
+    if not config_dir:
         return
 
-    logger.log('[x] PokemonGO Bot v1.0', 'green')
-    logger.log('[x] Configuration initialized', 'yellow')
+    kernel.set_config_file(config_dir)
+    kernel.boot()
 
     try:
-        bot = PokemonGoBot(config)
+        bot = kernel.container.get('pokemongo_bot')
```

Comment:
ah i had to change the `plugins.include` config key to a list of all directories to include to load pluginsfrom:

```yml
plugins:
    # Do not automatically load the specified plugins
    exclude:
        - socket

    # Plugin directories to load
    include:
        - ./plugins

    # Load plugin YAML config files from this folder
    config: "config/plugins"
```

i'll update the PR.